### PR TITLE
DOCS: Document pg_buffercache extension

### PR DESCRIPTION
**Pull request by VMWareJennyBeth**
Staging here: https://docs-staging.vmware.com/en/Greenplum-Database-7/GPDOCS-1218-pg_buffercache-extension/greenplum-database/GUID-ref_guide-modules-pg_buffercache.html

### CODE REVIEW

File: gpdb-doc/markdown/ref_guide/modules/intro.html.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ The following Greenplum Database and PostgreSQL `contrib` modules are installed;
 -   [orafce](orafce_ref.html) - Provides Greenplum Database-specific Oracle SQL compatibility functions.
 -   [pageinspect](pageinspect.html) - Provides functions for low level inspection of the contents of database pages; available to superusers only.
 -   [pg\_trgm](pg_trgm.html) - Provides functions and operators for determining the similarity of alphanumeric text based on trigram matching. The module also provides index operator classes that support fast searching for similar strings.
+-    [pg_buffercache](pg_buffercache.html) - Provides access to five views for obtaining cluster-wide shared buffer metrics.
```
Comment (bmdoil) [inline gpdb-doc/markdown/ref_guide/modules/intro.html.md:26]: nit: extra indent and underscore needs escape `[pg_buffercache]`

File: gpdb-doc/markdown/ref_guide/modules/pg_buffercache.html.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# pg_buffercache 
+
+The `pg_buffercache` module provides five views for obtaining cluster-wide shared buffer metrics:
+
+- `gp_buffercache`
+- `gp_buffercache_summary`
+-  `gp_buffercache_usage_counts`
```
Comment (bmdoil) [inline gpdb-doc/markdown/ref_guide/modules/pg_buffercache.html.md:7]: nit: extra indent

File: gpdb-doc/markdown/ref_guide/modules/pg_buffercache.html.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+- `gp_buffercache_summary_aggregate` 
+- `gp_buffercache_usage_counts_aggregate`
+
+The `gp_buffercache` view is a cluster-wide view that displays the `pg_buffercache` information from the coordinator and every primary segment for each buffer in the shared cache.
+
+The `gp_buffercache_summary` view is a cluster-wide view that displays the `pg_buffercache_summary` information from the coordinator and each primary segment, summarized as one row per segment. Similar and more detailed information is provided by `gp_buffercache`, but `gp_buffercache_summary` is significantly better performance-wise.
+
```
Comment (bmdoil) [inline gpdb-doc/markdown/ref_guide/modules/pg_buffercache.html.md:14]: The associated pg_* function should be a hyperlink to the postgres reference for the function if at all possible. We do the same for MPP-ized versions of PG views.

Comment (bmdoil) [inline gpdb-doc/markdown/ref_guide/modules/pg_buffercache.html.md:14]: The `gp_buffercache_usage_counts` view is a cluster-wide view that displays the `pg_puffercache_usage_counts` information from the coordinator and each primary segment returning summarized states of all shared buffers, aggregated over the possible usage count values. Similar and more detailed information is provided by `gp_buffercache`, but `gp_buffercache_usage_counts` is significantly better performance-wise.

File: gpdb-doc/markdown/ref_guide/modules/pg_buffercache.html.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# pg_buffercache 
+
+The `pg_buffercache` module provides five views for obtaining cluster-wide shared buffer metrics:
+
+- `gp_buffercache`
+- `gp_buffercache_summary`
+-  `gp_buffercache_usage_counts`
+- `gp_buffercache_summary_aggregate`
```
Comment (bmdoil) [inline gpdb-doc/markdown/ref_guide/modules/pg_buffercache.html.md:8]: gp_buffercache_summary_aggregated
gp_buffercache_usage_counts_aggregated

File: gpdb-doc/markdown/ref_guide/modules/pg_buffercache.html.md (hunk 3 above)
Comment (bmdoil) [inline gpdb-doc/markdown/ref_guide/modules/pg_buffercache.html.md:14]: The `gp_buffercache_summary_aggregated` view returns a single row with the aggregated output of `gp_buffercache_summary`

The `gp_buffercache_usage_counts_aggregated` view returns a single row with the aggregated output of `gp_buffercache_usage_counts`

File: gpdb-doc/markdown/ref_guide/modules/intro.html.md (hunk 1 above)
Comment (VMWareJennyBeth) [reply gpdb-doc/markdown/ref_guide/modules/intro.html.md:26]: Underscores were escaped automatically when we converted from DITA to Markdown. They're no-ops in the HTML. But I appreciate you calling it out.

File: gpdb-doc/markdown/ref_guide/modules/pg_buffercache.html.md (hunk 3 above)
Comment (VMWareJennyBeth) [reply gpdb-doc/markdown/ref_guide/modules/pg_buffercache.html.md:14]: Need clarification, have slacked you.

File: gpdb-doc/markdown/ref_guide/modules/pg_buffercache.html.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+- `gp_buffercache`
+- `gp_buffercache_summary`
+- `gp_buffercache_usage_counts`
+- `gp_buffercache_summary_aggregated` 
+- `gp_buffercache_usage_counts_aggregated`
+
+The `gp_buffercache` view is a cluster-wide view that displays the [`pg_buffercache`](https://www.postgresql.org/docs/current/pgbuffercache.html) information from the coordinator and every primary segment for each buffer in the shared cache.
+
+The `gp_buffercache_summary` view is a cluster-wide view that displays the [`pg_buffercache_summary`](https://www.postgresql.org/docs/16/pgbuffercache.html#PGBUFFERCACHE-SUMMARY) information from the coordinator and each primary segment, summarized as one row per segment. Similar and more detailed information is provided by `gp_buffercache`, but `gp_buffercache_summary` is significantly better performance-wise.z
```
Comment (bmdoil) [inline gpdb-doc/markdown/ref_guide/modules/pg_buffercache.html.md:13]: extra z at the end

**Review by bmdoil — APPROVED** [review]
LGTM

**Pull request merged**